### ISSUE TITLE
Support runlabels in containers.podman

### ISSUE THREAD

**Issue by pdostal** [containers/ansible-podman-collections#620]
Hello,

It would be nice to have [podman container runlabel](https://docs.podman.io/en/latest/markdown/podman-container-runlabel.1.html) module in `containers.podman`.

/kind feature

**Description**

Nowadays, we have the following:
```yaml
- name: Install all required parts for the container
  ansible.builtin.command: >-
    podman container runlabel install {{ container.image }}
  register: runlabel_install
  changed_when:
    - ('already exist' not in runlabel_install.stdout)
```

But that's not very pretty. I would appreciate something like:
```yaml
- name: Install all required parts for the container
  containers.podman.container_runlabel: install
```

With best regards
Pavel Dostál

**Comment by sshnaidm** [comment]
Hi, Pavel
I wonder if such module can be idempotent, or it's like `shell` module - "shoot and go".
Seems like it should be two modules, one for `display` runlabel, and one for execution.

**Comment by pdostal** [comment]
Hello Sergey,
 * for listing the image runlabels and getting their values I believe `containers.podman.podman_image_info` can be reused.
 * but running the image runlabels is another story. Of course it can be done via `ansible.builtin.shell` but it's from my opinion not very elegant.

**Comment by sshnaidm** [comment]
If we don't need to ensure idempotency (which I don't see how can be implied here), then it should be pretty easy. @pdostal would you like to submit a patch? You can take [`podman_tag`](https://github.com/containers/ansible-podman-collections/blob/master/plugins/modules/podman_tag.py) as a reference, or [`podman_login`](https://github.com/containers/ansible-podman-collections/blob/master/plugins/modules/podman_login.py), [`podman_save`](https://github.com/containers/ansible-podman-collections/blob/master/plugins/modules/podman_save.py).

**Comment by sshnaidm** [comment]
Fixed by #626
